### PR TITLE
Add explanation of embeddings

### PR DESCRIPTION
**Pull request by InAnYan**

### CODE REVIEW

**Review by koppor — CHANGES_REQUESTED** [review]
Please no "it's", always long form.

Please: One sentence per line. (In hope that GitBook does that too and does not crash this)

**Comment by InAnYan** [comment]
Updated

**Comment by koppor** [comment]
Updated too. @InAnYan I hope, you like.

File: en/ai/README.md [hunk 1]
```diff
@@ -27,17 +27,15 @@ In this window you can see the following elements:
...
-In order to use it you need to configure JabRef to use your API key.
-
-JabRef processes linked files this way: the file is split into parts of fixed-length (also called *chunks*), and then an *embedding* is generated.
-Embedding is a representation of a part of text.
-It is a vector that represents the meaning of the text.
-This vector has a crucial property: texts with similar meaning have vectors that are close to (this is called *vector similarity*).
-So, whenever you ask AI a question, JabRef tries to find relevant pieces of text from the indexed files using vector similarity.
+JabRef uses external AI providers to do the actual work.
+You can choose between OpenAI, Mistral AI, and Hugging Face.
+They all run "Large Language Models" (LLMs) to process the quests.
```
Comment (InAnYan) [inline en/ai/README.md:32]: quests :)

**Comment by InAnYan** [comment]
Updated, ready to merge

**Pull request merged**